apparmor.d/groups/virt/k3s

```diff
@@ -26,7 +27,7 @@ profile k3s @{exec_path} {
   capability sys_resource,
 
   ptrace peer=@{profile_name},
-  ptrace (read) peer={cri-containerd.apparmor.d,cni-xtables-nft,kubernetes-pause,mount,unconfined},
```

Comment:
Just figuring this out. What is `cri-containerd.apparmor.d` ?

Reply:
Every container runtime has it's own default AppArmor profile. If you don't specify one (and don't run a privileged container) the default is applied. You can see the default template here: https://github.com/containerd/containerd/blob/main/contrib/apparmor/template.go#L43